theano/sandbox/gpuarray/conv_desc.c

```diff
@@ -0,0 +1,34 @@
+#section support_code_apply
+
+int conv_desc(PyArrayObject *filt_shp,
+              cudnnConvolutionDescriptor_t *desc) {
```

Comment:
I'm confused. Is that function part of the support code for this apply node, or is it the actual function that will perform the thunk?